src/jepsen/voltdb/runner.clj

```diff
@@ -137,9 +141,14 @@
                    :nodes     (:nodes test)
                    :faults    (:nemesis opts)
                    ; TODO: add support for targeting primaries
```

Comment:
Add comment stating why we want to kill 1 node at a time

Reply:
I think killing a majority of nodes is wrong test. Kyle's understanding was that he wanted to break "majority" ring. VoltDB works differently. As long as a single node for a partition remains, Voltdb works. Otherwise, Voltdb is shut.
We might want to do the following
- use "target" option to indicate if nemesis should kill "all" or one.
- develope a new nemesis that can kill a given number of nodes. It is not available out of box.